src/components/fields/date-field/date-field.tsx

```diff
@@ -26,7 +26,14 @@ export const DateField = (props: IGenericFieldProps<IDateFieldSchema>) => {
 		id,
 		isDirty,
 		onChange,
-		schema: { label: _label, useCurrentDate, dateFormat = DEFAULT_DATE_FORMAT, validation, ...otherSchema },
+		schema: {
+			label: _label,
+			useCurrentDate,
+			dateFormat = DEFAULT_DATE_FORMAT,
+			validation,
+			hideInputKeyboard,
```

Comment:
no need to extract `hideInputKeyboard`, you can leave it in `otherSchema` and it will be applied to `Form.DateInput`